### PR TITLE
Remove 'use multiple accounts' doc 

### PR DESCRIPTION
**Pull request by zuluecho9**
This was based on knowing that this 'use multiple accounts' doc was woefully out of date and not helping anyone, and then we did the work based on recent https://github.com/newrelic/docs-website/issues/1555 complaint. Also added a little work clarifying how account mgmt works in new paradigm.

### CODE REVIEW

File: src/content/docs/accounts/accounts-billing/account-structure/mastersub-account-structure.mdx [hunk 1]
```diff
@@ -16,15 +16,27 @@ redirects:
...
+  - /docs/accounts-partnerships/accounts/account-setup/create-multiple-accounts
+  - /docs/accounts-partnerships/accounts/account-setup/use-multiple-accounts
+  - /docs/accounts-partnerships/install-new-relic/account-setup/use-multiple-accounts  
 ---
 
 If you have Pro or Enterprise tier, you can create a more complex account structure with multiple accounts in one organization.
 
+## Requirements [#requirements]
+
+How you manage accounts and users differs based on which account/user model your organization is on. To learn more about the account/user structures, see [User models](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model/#user-models). The major differences are:
```
Comment (paperclypse) [inline src/content/docs/accounts/accounts-billing/account-structure/mastersub-account-structure.mdx:30]: Although the User models link seems to work, it does look like it's formatted oddly. Should it be: `/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model#user-models`

File: src/content/docs/accounts/accounts-billing/account-structure/mastersub-account-structure.mdx [hunk 2]
```diff
@@ -16,15 +16,27 @@ redirects:
...
+  - /docs/accounts-partnerships/install-new-relic/account-setup/use-multiple-accounts  
 ---
 
 If you have Pro or Enterprise tier, you can create a more complex account structure with multiple accounts in one organization.
 
+## Requirements [#requirements]
+
+How you manage accounts and users differs based on which account/user model your organization is on. To learn more about the account/user structures, see [User models](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model/#user-models). The major differences are: 
+
+* New Relic One user model: The Standard pricing tier includes only one account per organization, and higher tiers allow you to have multiple accounts. To add more accounts to your organization, currently you must be helped by a New Relic representative. To add users to accounts, see [Manage users](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#understand-concepts).
```
Comment (paperclypse) [inline src/content/docs/accounts/accounts-billing/account-structure/mastersub-account-structure.mdx:32]: See previous comment re. the Manage users link

**Review by paperclypse — APPROVED** [review]
Looks good. Not sure if my comments are worth addressing or not...

File: src/content/docs/accounts/accounts-billing/account-structure/mastersub-account-structure.mdx (hunk 1 above)
Comment (zuluecho9) [reply src/content/docs/accounts/accounts-billing/account-structure/mastersub-account-structure.mdx:30]: are you referring to the / or lack of it before anchor link? from my undrestanding that works either way.

Comment (paperclypse) [reply src/content/docs/accounts/accounts-billing/account-structure/mastersub-account-structure.mdx:30]: the / before the anchor link. It just looks weird to me, but it worked when I checked it so it's probably all right.

**Pull request merged**